website/docs/api/plugins/plugin-content-blog.mdx

```diff
@@ -48,6 +48,7 @@ Accepted fields:
 | `blogSidebarTitle` | `string` | `'Recent posts'` | Title of the blog sidebar. |
 | `routeBasePath` | `string` | `'blog'` | URL route for the blog section of your site. **DO NOT** include a trailing slash. Use `/` to put the blog at root path. |
 | `tagsBasePath` | `string` | `'tags'` | URL route for the tags section of your blog. Will be appended to `routeBasePath`. **DO NOT** include a trailing slash. |
+| `pageBasePath` | `string` | `'page'` | URL route for the pages section of your blog. Will be appended to `routeBasePath`. **DO NOT** include a trailing slash. |
```

Comment:
```suggestion
| `tagsBasePath` | `string` | `'tags'` | URL route for the tags section of your blog. Will be appended to `routeBasePath`. |
| `pageBasePath` | `string` | `'page'` | URL route for the pages section of your blog. Will be appended to `routeBasePath`. |
```

I'd prefer to remove this entirely and make our code less sensitive to the presence of a leading/trailing slashes in those params.

Of course it's not needed for this PR but it would be a nice little cleanup at the same time.

Reply:
Removed.